common/src/config/GameConfig.cpp

```diff
@@ -165,6 +165,7 @@ bool GameConfig::visit_vars(T&& visitor, bool is_save)
     result &= visitor(dash_faction_key, "Direct Input", direct_input);
     result &= visitor(dash_faction_key, "Fast Start", fast_start);
     result &= visitor(dash_faction_key, "Scoreboard Animations", scoreboard_anim);
+    result &= visitor(dash_faction_key, "Minimal Spectate UI", spectate_mode_minimal_ui);
```

Comment:
Consistency would be better imo.
```suggestion
    result &= visitor(dash_faction_key, "Spectate Mode Minimal UI", spectate_mode_minimal_ui);
```